plugins/DevicesDetection/templates/detection.twig

```diff
@@ -19,7 +19,7 @@
                     maxHeight: 400
                 });
             });
-            ajaxHandler.send(true);
+            ajaxHandler.send();
```

Comment:
Here we might need to save a reference to a pending ajax request and if there is one, abort this request before requesting a new one. Otherwise it can come result in funny results when clicking on a list, then on another list and the first XHR finishes after the second XHR